Dockerfile.fhir_resource_validator

```diff
@@ -0,0 +1,12 @@
+FROM markiantorno/validator-wrapper
+
+USER root
+# Java certs need to be installed as root, so switch to that user for this step only
+# (the image does not contain 'sudo')
+RUN wget https://gitlab.mitre.org/mitre-scripts/mitre-pki/-/raw/master/tool_scripts/install_certs.sh -O - | MODE=java sh
+
+USER $APPLICATION_USER
+
+# CMD should be the same as in the original validator-wrapper Dockerfile
+# https://github.com/hapifhir/org.hl7.fhir.validator-wrapper/blob/master/Dockerfile
+CMD ["java", "-server", "-XX:+UnlockExperimentalVMOptions", "-XX:InitialRAMPercentage=79", "-XX:MinRAMPercentage=79", "-XX:MaxRAMPercentage=79", "-XX:+UseG1GC", "-XX:MaxGCPauseMillis=100", "-XX:+UseStringDeduplication", "-XX:+CrashOnOutOfMemoryError", "-jar", "validator-wrapper.jar", "-startServer"]
```

Comment:
It's unfortunate that we have to repeat the `CMD`.

Reply:
I'm taking a closer look at this now, I may have misunderstood some stackoverflow answers yesterday. We may not need to specify a CMD at all and it will inherit the parent image CMD.